lib/rubocop/cop/lint/debugger.rb

```diff
@@ -33,35 +34,14 @@ module Lint
       #     do_something
       #   end
       class Debugger < Base
-        MSG = 'Remove debugger entry point `%<source>s`.'
-
-        RESTRICT_ON_SEND = %i[
```

Comment:
You should keep this constant, as it's special for RuboCop (it limits the `on_send` callbacks). Just initialize it with the data from the config.

Reply:
The problem if I hardcode the values from the configuration in this constant is not recognizing new values from the config.

This test, for example, start failing: https://github.com/rubocop-hq/rubocop/blob/12bd06882eaca12bd33c77f31e8d976bc3ad0bed/spec/rubocop/cop/lint/debugger_spec.rb#L15-L20

What I did, for now, is initializing the constant but with an empty array.

Reply:
Ah, yeah - that's a fair point.